_includes/coffee/result_comparison.coffee

```diff
@@ -207,22 +207,24 @@ func_xy = (func) ->
 
 
 reorder = curry(
-  (name, datum_x, datum_y, values) ->
+  (center, name, datum_x, datum_y, values) ->
     ### Reorder a set of of data points based on the polar angle.
 
     Args:
+      center: offset the center value for defining the angle
       name: the name to match with the name field in the list of data
         items (e.g 'free_energy')
-      datum_x: the subfied of x data to extract (e.g. 'x')
-      datum_y: the subfied of y data to extract (e.g. 'y')
+      datum_x: the subfield of x data to extract (e.g. 'x')
+      datum_y: the subfield of y data to extract (e.g. 'y')
       values: all the data values
 
     Returns:
       the calculated values as [x_values, y_values]
 
     ###
     calc_theta = sequence(
```

Comment:
Use of `x` for the point name makes for confusing reading. Please consider a different character:

- `r` or `R` for real?
- `c` or `X` for coordinate?
- `p` or `P` for point?

Reply:
Ooh, `data` is very nice.

Reply:
Fixed with fdf7aecf. Please rereview.